Functions/xASL_im_MaskPeakVascularSignal.m

```diff
@@ -70,8 +75,8 @@
 		warning('No usable M0 image, using CBF only');
 		M0_im = 1;
 	else
-		M0_im = xASL_io_Nifti2Im(Path_M0);
-		if ~(min(size(TreatedCBF)==size(M0_im))) % if CBF & M0 images arent the same size (which is why rM0 should be the input path in case of different sizes)
+		M0_im = xASL_io_Nifti2Im(Path_M0); % allow both a path or image matrix
+		if ~(min(size(PWI)==size(M0_im))) % if CBF & M0 images arent the same size (which is why rM0 should be the input path in case of different sizes)
```

Comment:
Small detail: replace CBF with PWI in comments?

Reply:
Done.